scripts/msct_register.py

```diff
@@ -119,17 +119,14 @@ def register2d_centermassrot(fname_src, fname_dest, fname_src_seg=None, fname_de
         fname_dest_seg : name of the segmentation of the destination image
         fname_warp: name of output 3d forward warping field
         fname_warp_inv: name of output 3d inverse warping field
-        rot: estimate rotation with PCA (type: int)
+        rot: estimate rotation with PCA (=1), HOG (=2)
```

Comment:
you also need to explicit value 0.  E.g.: 
~~~
No rotation (=0), rotation estimated with PCA (=1) or with HOG (=2). Default: XXX
~~~
For XXX: grab the default value and display it here.